test/jwt_test.cc

```diff
@@ -520,27 +521,31 @@ TEST(JwtParseTest, GoodNestedJwt) {
    *      "key-3": true,
    *      "key-4": 9999
    *    }
-   *  }
+   *  },
```

Comment:
Can you add assertions to the test to verify these are accessible from the `Jwt` object?

Also suggest you add a few corner cases to this token, such as URL with escaped `/` characters, malformed URLs, etc.

Reply:
Haha, I went to a lot of effort to get the JWT token updated with the new claims and then somehow managed to not commit the tests that used it.

Added standard happy-path tests for now